src/common/base-audit.js

```diff
@@ -100,6 +100,18 @@ export async function noopUrlResolver(site) {
   return site.getBaseURL();
 }
 
+export async function defaultJobProvider(jobId, context) {
```

Comment:
seems wrong at this location as only audits that are based on the async job audit can leverage this provider.